source/localizable/smart_cart/webhook.html.md.erb

```diff
@@ -556,6 +561,7 @@ Name           | Type   | Value | Description
     "comments": "Παράδοση στο γραφείο",
     "courier": "ACS",
     "courier_voucher": "https://b.scdn.gr/path-to/voucher.pdf",
+    "courier_tracking_codes": ["123456789", "123456780"],
```

Comment:
You have added 2 tracking codes here, but the corresponding `changes > courier_tracking_codes` object includes only a single code. Feel free to either remove the last one `123456780` from here, or append it in the changes object.

Reply:
https://github.com/skroutz/developer.skroutz.gr/blob/marketplace/add_tracking_codes_to_order/source/localizable/smart_cart/webhook.html.md.erb#L597

Reply:
since this is an ACS order, and we currently do not support multiple parcels from this company I will keep just one here